pandas/core/window.py

```diff
@@ -1354,7 +1354,9 @@ def _get_center_of_mass(com, span, halflife, alpha):
 
 
 def _offset(window, center):
-    if not com.is_integer(window):
+    if com.is_float(window):
```

Comment:
@jreback So ```pd.DataFrame(np.arange(10)).rolling(2.).median()``` should work, while ```pd.DataFrame(np.arange(10)).rolling(2., center=True).median()``` should fail with argument validation (only difference is ```center=True```)? Can you confirm?